home/ivp-planning/advanced-tools/path-tool.md

```diff
@@ -58,15 +58,15 @@ The length and angle information of each path can be turned on and off individua
 
 ![](../../../.gitbook/assets/iVP\_path\_tool\_draw\_path\_right\_click\_point\_menu.jpg)
 
-Alternatively it is possible to hide or show all lengths or angles by **right clicking** on the path entry in the [path panel](../user-interface/path-panel.md) and subsequently clicking on **"Edge Lengths"** or **"Angles"**.
+Alternatively it is possible to hide or show all lengths or angles by changing the entry in the measurements of the [properties](../user-interface/the-info-panel.md) and subsequently clicking on **"Edge Lengths"** or **"Angles"**.
 
 ![](../../../.gitbook/assets/iVP\_path\_tool\_draw\_path\_path\_panel.jpg)
 
 ![](../../../.gitbook/assets/iVP\_path\_tool\_list\_right\_click\_menu\_measurements.jpg)
 
 ## Area measurements:
 
-Besides lengths and angles, areas are measurable as well. To activate this option, **right click on the path entry** in the path panel, followed by a click in **"Area"**.
+Besides lengths and angles, areas are measurable as well. To activate this option, **right click on the path entry** in the measurements tab of the [properties](../user-interface/the-info-panel.md), followed by a click in **"Area"**.
```

Comment:
There is no right click action in the properties panel

Reply:
Sorry, i did not check the text thoroughly as I will rework the whole section soon. Changed the information to be "To activate this option, **check the 'Area' box** under the measurements tab of the..."